test/e2e/e2e.go

```diff
@@ -584,6 +606,74 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 		framework.ExpectNoError(err, "one or more nodes failed to go back ready, schedulable, and untainted")
 	})
 
+	ginkgo.FIt("should provide Internet connection continuously when all pods are killed on node running master instance of ovnkube-master.", func() {
+		ginkgo.By(fmt.Sprintf("Running container which tries to connect to %s in a loop", extDNSIP))
+
+		ovnKubeMasterNode, err := findOvnKubeMasterNode()
+		framework.ExpectNoError(err, fmt.Sprintf("unable to find current master of ovnkuber-master cluster %v", err))
+
+		podChan, errChan := make(chan *v1.Pod), make(chan error)
+		go func() {
+			defer ginkgo.GinkgoRecover()
+			checkContinuousConnectivity(f, "", "connectivity-test-continuous", extDNSIP, 53, 30, 30, podChan, errChan)
+		}()
+
+		testPod := <-podChan
```

Comment:
i think you need to read from the errChan first before you read from the podChan according to the checkContinuousConnectivity text